### PR TITLE
Update metadata in package.json to use default commands & Update README

### LINKED ISSUE #2336: Document algolia configuration
Algolia is our search engine that's built into Docusaurus, the configuration for the indexing that it does for faster lookup is in another repository that's maintained by them. 
Put something into the README.md to explain how to access and modify it.

### PR DESCRIPTION
**Pull request by salmad3**
- To use commands "polkadot:" & "kusama:" for new version instead of "polkadot2:" & "kusama2:"
- Reference: https://github.com/w3f/polkadot-wiki/issues/2294#issuecomment-880851482
---
README updates 
- resolves #2336 
- The added images attempt to make the page more attractive - links to both wikis and then some colours after that. 
- The description for Algolia should be okay but a good idea to double check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,20 +50,50 @@ run:
...
-files, not the MD files. The value placeholders are defined in
-[scripts/inject-dict.json](scripts/inject-dict.json). The placeholders should be included in text
-surrounded by double curly braces, like so: `{{ num_validators }}`.
+static files, not the MD files. The value placeholders are defined in the `inject-dict.json` 
+file located in `scripts`. 
+
+The placeholders should be included in text surrounded by double curly braces, 
+like so: `{{ num_validators }}`.
 
 To use the replace script:
```
Comment (nam-hle) [inline README.md]: We dont have the scripts for injection anymore

File: package.json [hunk 2]
```diff
@@ -24,26 +24,26 @@
...
+    "polkadot:pdf": "yarn pdf --initialDocURLs \"http://localhost:3000/docs/getting-started\" --outputPDFFilename=\"website-v2/static/polkadot-wiki.pdf\" --coverTitle=\"POLKADOT WIKI\"",
+    "polkadot:crowdin:upload:source": "crowdin upload sources --config ./crowdin2.yaml -b master",
+    "polkadot:crowdin:download:ru": "crowdin download --config ./crowdin2.yaml -b master -l ru",
+    "polkadot:crowdin:download:zh-CN": "crowdin download --config ./crowdin2.yaml -b master -l zh-CN",
+    "polkadot:copy-assets:ru": "shx mkdir -p website-v2/i18n/ru-RU/docusaurus-plugin-content-docs/current; shx cp -R docs/assets website-v2/i18n/ru-RU/docusaurus-plugin-content-docs/current",
+    "polkadot:copy-assets:zh-CN": "shx mkdir -p website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current; shx cp -R docs/assets website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current",
     "pdf": "npx mr-pdf --contentSelector=\"article\" --paginationSelector=\".pagination-nav__item--next > a\" --pdfFormat=\"A4\" --pdfMargin=\"90,60,80,60\"",
     "fetch": "node --experimental-json-modules scripts/fetch.mjs --isPolkadot",
     "mirror2": "node --experimental-json-modules scripts/mirror2.mjs",
```
Comment (nam-hle) [inline package.json]: Please consider to change the script name too.

**Review by nam-hle — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (nam-hle) [reply package.json]: Perhaps we also need to change script file names.

Comment (salmad3) [reply package.json]: Updated

File: README.md (hunk 1 above)
Comment (salmad3) [reply README.md]: Updated

**Comment by jakehemmerle** [comment]
when there are multiple features, please split them into seperate prs. we can usually away with two small features but really shouldn't be 3 or more as they become super hard to digest and go through in one go. you also included some stale related stuff that you did not document

only suggestion I have might be to rename `crowdin1.yaml` to something like `crowdin-docs-v1.yaml` so it's a little more descriptive (I do recognize that it's been done this way for a bit now)

otherwise lgtm!

**Review by jakehemmerle — APPROVED** [review]

**Review by DrW3RK — APPROVED** [review]
Looks good! Although, we need to implement polkadot: inject and kusama: inject for the latest version of wiki

**Comment by salmad3** [comment]
> Looks good! Although, we need to implement polkadot: inject and kusama: inject for the latest version of wiki

Gotcha

**Review by jakehemmerle — APPROVED** [review]

**Comment by jakehemmerle** [comment]
@DannyS03 don't merge, you forgot to change all references of `crowdin1.yaml` to the new name

**Comment by salmad3** [comment]
@jakehemmerle updated, okay to merge?

**Comment by jakehemmerle** [comment]
@DannyS03 yup looks good

**Pull request merged**